src/appmake/c7420.c

```diff
@@ -179,8 +176,8 @@ int c7420_exec(char* target)
         /****************/
         /* BASIC loader */
         /****************/
-        sprintf(lsbbuf, "%i", (int)pos % 256); /* no more than 3 characters long */
-        sprintf(msbbuf, "%i", (int)pos / 256); /* no more than 3 characters long */
+        sprintf(lsbbuf, "%i", (uint8_t) (pos % 256) ); /* no more than 3 characters long */
+        sprintf(msbbuf, "%i", (uint8_t) (pos / 256) ); /* no more than 3 characters long */
```

Comment:
This change in type cast doesn't change anything, since both numbers are at most 3 characters long (values 0..255) which fits exactly in uint8_T.